src/models/ITestResult.ts

```diff
@@ -50,6 +50,7 @@ export interface ITestResult {
   make?: string;
   model?: string;
   bodyType?: BodyTypeModel;
+  reapplicationDate?: string;
```

Comment:
This is at the wrong level, it should be within the test type model (it's the IVA version of the expiry date).  It should also be a date

Reply:
To be fair in a TS model dates are strings for us.

Reply:
Our other dates are ...

`${key} : string | Date;`

So if we can keep it consistent with that, that will probably be safest